Dockerfile

```diff
@@ -14,21 +14,26 @@ EXPOSE 8000
 
 # Install operating system dependencies.
 RUN apt-get update -y && \
-    apt-get install -y apt-transport-https rsync libmagickwand-dev && \
+    apt-get install -y apt-transport-https rsync libmagickwand-dev unzip postgresql-client-13 && \
     rm -rf /var/lib/apt/lists/*
 
+# Don't use the root user.
+ARG UID=1000
+RUN useradd wagtailio -u $UID
```

Comment:
Suggestion: It might be worth adding `-m`/`--create-home` here to allow people to install their own stuff in the container more easily and add things like `.bashrc` files for customisation.

Reply:
Added, thanks!